target_goal:
  t_01: {pose: [23.692, -25.405, 0.0], orientation: [0.0, 0.0, 0.762, 0.646]}
  t_02: {pose: [2.328, 0.210, 0.0], orientation: [0.0, 0.0, -0.683, 0.730]}
  t_03: {pose: [2.328, 0.210, 0.0], orientation: [0.0, 0.0, -0.683, 0.730]}
  btn_01: {pose: [5.296, -2.623, 0.0], orientation: [0.0, 0.0, 0.754, 0.656]}
  btn_02: {pose: [4.277, -5.056, 0.0], orientation: [0.0, 0.0, 0.465, 0.885]}
  btn_03: {pose: [4.277, -5.056, 0.0], orientation: [0.0, 0.0, 0.465, 0.885]}
  _standby: {pose: [24.968, -22.960, 0.0], orientation: [0.0, 0.0, -0.996,0.083]}

